codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala

```diff
@@ -54,7 +65,9 @@ class SerializerCheckCompilerPluginComponent(
           }
           .map(implDefAnnotationsTuple =>
             (implDefAnnotationsTuple._1, implDefAnnotationsTuple._2.filter(_.tpe.toString == serializerType)))
-          .filter(_._2.nonEmpty)
+          .filter(
+            _._2.nonEmpty // as we are interested in checking only these types, where @Serializer annotation has been used
+          )
```

Comment:
While you're here... pls decompose tuples explicitly so as not to rely on `_1`, `_2`